pkgs/by-name/xd/xdg-desktop-portal-termfilechooser/package.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, meson
+, ninja
+, pkg-config
+, wayland-protocols
+, wayland-scanner
+, inih
+, libdrm
+, mesa
+, scdoc
+, systemd
+, wayland
+}:
+stdenv.mkDerivation {
+  pname = "xdg-desktop-portal-termfilechooser";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "nani8ot";
+    repo = "xdg-desktop-portal-termfilechooser";
+    rev = "main";
```

Comment:
`main` is a moving target. Use raw git hash instead - or some tag like `v${version}`.